src/swagger/index.js

```diff
@@ -42,6 +42,8 @@ class Swagger {
   }
 
   getPath (url) {
+    // The / at the end will match successfully in the express routing rule base
+    url = url.replace(new RegExp(`${this.basePathPrefix}/{1,2}`), `${this.basePathPrefix}/`)
```

Comment:
why not use `url.replace(`${this.basePathPrefix}//`, `${this.basePathPrefix}/`)`?